File: prime.swift
var primes : Int[] = []
for i in 2..1 {
  var isPrime = true
  for p in primes {
    if i % p == 0 {
      isPrime = false
      break
    }
  }
  if isPrime {
    println(i)
    primes.append(i)
  }
}

File: prime.rb
#!/usr/bin/env ruby
primes = []
num = 2
while true
  is_prime = true
  for p in primes
    if num % p == 0
      is_prime = false
      break
    end
  end
  if is_prime
    primes.push num
    printf "%d ", num
  end
  num += 1
end
